jupyter-tensorflow-notebook/Dockerfile

```diff
@@ -1,28 +1,21 @@
-  
-FROM jupyter/tensorflow-notebook:python-3.8.8
-# Get the latest image tag at:
-# https://hub.docker.com/r/jupyter/minimal-notebook/tags/
-# Inspect the Dockerfile at:
-# https://github.com/jupyter/docker-stacks/tree/HEAD/minimal-notebook/Dockerfile
+
+FROM jupyter/tensorflow-notebook:ubuntu-22.04
 
 # Switch to root to install packages
 USER root
 
-# install additional package...
-RUN apt-get  --allow-unauthenticated update --yes && apt-get --allow-unauthenticated install --yes  gnupg
-RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/7fa2af80.pub 
-RUN apt-key add 7fa2af80.pub
-RUN echo "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /" >> /etc/apt/sources.list
-RUN apt-get remove --yes nvidia* *cuda*
-#RUN apt-get  --allow-unauthenticated update --yes && apt-get --allow-unauthenticated install --yes  curl wget patch libcudnn8 cuda-11-1 nvidia-driver-470
-RUN apt-get  --allow-unauthenticated update --yes && apt-get --allow-unauthenticated install --yes  curl wget patch cuda-toolkit-11-1 libcudnn8
-# install tensorflow for GPUs
-RUN pip uninstall tensorflow tensorboard --yes
-RUN pip install tensorflow-gpu tensorboard
-RUN pip install jupyter-server-proxy
-RUN pip install jupyterlab --upgrade
-
+RUN apt-get update --yes \
+    # Nvidia components
+    && apt-get install --yes nvidia-cuda-toolkit nvidia-cudnn \
+    # User facing components
+    && apt-get install --yes curl wget patch \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Unsure why we want to remove this file but it won't break anything

Reply:
So it's [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments), otherwise we effectively bundle cached .debs into the docker image

Reply:
aah good point, thanks for clarifying